.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: ci
+
+on:
+  pull_request:
+    branches:
+      - main
+
+  push:
+    branches:
+      - main
+
+  workflow_dispatch:
```

Comment:
what is this line? it is bare which I find suspicious.

Reply:
It's legit. It allows a GitHub workflow to be triggered manually and that workflow does not require any inputs.

Reply:
haha. it's to trigger manually

Reply:
oops just saw your comment @mdmower. long time, hi!!

Reply:
Hey Matt! Thanks for the info & hope all is well. Feel free to drive by with helpful comments whenever you like 😄